types/api/v1beta1/event_data.proto

```diff
@@ -0,0 +1,145 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/wrappers.proto";
+
+message EventValue {
+    oneof value {
+        google.protobuf.Int32Value int32 = 1; // intT
+        google.protobuf.Int64Value int64 = 2; // longT
+        google.protobuf.UInt32Value uint32 = 3; // uintT, modeT, devT
+        google.protobuf.UInt64Value uint64 = 4; // ulongT, offT, sizeT, pointerT
+        google.protobuf.StringValue string = 5; // strT
+        google.protobuf.BytesValue bytes = 6; // bytesT
+        google.protobuf.BoolValue bool = 7; // boolT
+        UInt8Value uint8 = 8; // u8T
+        UInt16Value uint16 = 9; // u16T
+        StringArrayValue string_array = 10; // strArrT, argsArrT
+        IntArrayValue int_array = 11; // intArr2T
+        UInt64ArrayValue uint64_array = 12; // uint64ArrT
+        SockAddrValue sockaddr = 13; // sockaddrT
+        CredValue cred = 14; // credT
+        TimespecValue timespec = 15; // timespecT
+        SyscallArgsValue syscall_args = 16;
+        TriggerEvent trigger_event = 17;
+    }
+}
+
+message TriggerEvent {
+    uint32 id = 1;
+    string name = 2;
+    // TODO: is this correct?
+    map<string, EventValue> data = 3;
```

Comment:
Mostly yes, maybe with the exception that TriggerEvent can't contain another TriggerEvent